kommunicate/src/main/java/io/kommunicate/callbacks/KmStartConversationHandler.java

```diff
@@ -1,6 +1,13 @@
 package io.kommunicate.callbacks;
 
-import com.applozic.mobicomkit.uiwidgets.async.AlChannelCreateAsyncTask;
 
-public interface KmStartConversationHandler extends AlChannelCreateAsyncTask.TaskListenerInterface {
+import android.content.Context;
+
+import com.applozic.mobicomkit.feed.ChannelFeedApiResponse;
+import com.applozic.mobicommons.people.channel.Channel;
+
+public interface KmStartConversationHandler {
```

Comment:
@reytum is it similar to KMStartChatHandler? can both use the same handler or it has to be different?

Reply:
The KmStartChatHandler is deprecated and will be removed. Both are different and used by different codes.